screenshots/Readme.md

```diff
@@ -0,0 +1,5 @@
+## Contributers
```

Comment:
I propose you set up `CONTRIBUTERS.md` much like we did for other repos (in the repo root).